config/replacements.php

```diff
@@ -151,112 +151,138 @@
-    'n\\ZendOAuth\\' => 'n\\ZendOAuth\\',
-    'o\\ZendOAuth\\' => 'o\\ZendOAuth\\',
-    'p\\ZendOAuth\\' => 'p\\ZendOAuth\\',
-    'q\\ZendOAuth\\' => 'q\\ZendOAuth\\',
-    'r\\ZendOAuth\\' => 'r\\ZendOAuth\\',
-    's\\ZendOAuth\\' => 's\\ZendOAuth\\',
-    't\\ZendOAuth\\' => 't\\ZendOAuth\\',
-    'u\\ZendOAuth\\' => 'u\\ZendOAuth\\',
-    'v\\ZendOAuth\\' => 'v\\ZendOAuth\\',
-    'w\\ZendOAuth\\' => 'w\\ZendOAuth\\',
-    'x\\ZendOAuth\\' => 'x\\ZendOAuth\\',
-    'y\\ZendOAuth\\' => 'y\\ZendOAuth\\',
-    'z\\ZendOAuth\\' => 'z\\ZendOAuth\\',
-    'a\\ZendService\\' => 'a\\ZendService\\',
-    'b\\ZendService\\' => 'b\\ZendService\\',
-    'c\\ZendService\\' => 'c\\ZendService\\',
-    'd\\ZendService\\' => 'd\\ZendService\\',
-    'e\\ZendService\\' => 'e\\ZendService\\',
-    'f\\ZendService\\' => 'f\\ZendService\\',
-    'g\\ZendService\\' => 'g\\ZendService\\',
-    'h\\ZendService\\' => 'h\\ZendService\\',
-    'i\\ZendService\\' => 'i\\ZendService\\',
-    'j\\ZendService\\' => 'j\\ZendService\\',
-    'k\\ZendService\\' => 'k\\ZendService\\',
-    'l\\ZendService\\' => 'l\\ZendService\\',
-    'm\\ZendService\\' => 'm\\ZendService\\',
-    'n\\ZendService\\' => 'n\\ZendService\\',
-    'o\\ZendService\\' => 'o\\ZendService\\',
-    'p\\ZendService\\' => 'p\\ZendService\\',
-    'q\\ZendService\\' => 'q\\ZendService\\',
-    'r\\ZendService\\' => 'r\\ZendService\\',
-    's\\ZendService\\' => 's\\ZendService\\',
-    't\\ZendService\\' => 't\\ZendService\\',
-    'u\\ZendService\\' => 'u\\ZendService\\',
-    'v\\ZendService\\' => 'v\\ZendService\\',
-    'w\\ZendService\\' => 'w\\ZendService\\',
-    'x\\ZendService\\' => 'x\\ZendService\\',
-    'y\\ZendService\\' => 'y\\ZendService\\',
-    'z\\ZendService\\' => 'z\\ZendService\\',
-    'a\\ZendXml\\' => 'a\\ZendXml\\',
-    'b\\ZendXml\\' => 'b\\ZendXml\\',
-    'c\\ZendXml\\' => 'c\\ZendXml\\',
-    'd\\ZendXml\\' => 'd\\ZendXml\\',
-    'e\\ZendXml\\' => 'e\\ZendXml\\',
-    'f\\ZendXml\\' => 'f\\ZendXml\\',
-    'g\\ZendXml\\' => 'g\\ZendXml\\',
-    'h\\ZendXml\\' => 'h\\ZendXml\\',
-    'i\\ZendXml\\' => 'i\\ZendXml\\',
-    'j\\ZendXml\\' => 'j\\ZendXml\\',
-    'k\\ZendXml\\' => 'k\\ZendXml\\',
-    'l\\ZendXml\\' => 'l\\ZendXml\\',
-    'm\\ZendXml\\' => 'm\\ZendXml\\',
-    'n\\ZendXml\\' => 'n\\ZendXml\\',
-    'o\\ZendXml\\' => 'o\\ZendXml\\',
-    'p\\ZendXml\\' => 'p\\ZendXml\\',
-    'q\\ZendXml\\' => 'q\\ZendXml\\',
-    'r\\ZendXml\\' => 'r\\ZendXml\\',
-    's\\ZendXml\\' => 's\\ZendXml\\',
-    't\\ZendXml\\' => 't\\ZendXml\\',
-    'u\\ZendXml\\' => 'u\\ZendXml\\',
-    'v\\ZendXml\\' => 'v\\ZendXml\\',
-    'w\\ZendXml\\' => 'w\\ZendXml\\',
-    'x\\ZendXml\\' => 'x\\ZendXml\\',
-    'y\\ZendXml\\' => 'y\\ZendXml\\',
-    'z\\ZendXml\\' => 'z\\ZendXml\\',
+    // class ZendAcl in zend-expressive-authorization-acl
+    'ZendAcl' => 'LaminasAcl',
+    'Acl\\ZendAcl' => 'Acl\\LaminasAcl',
+    'Acl\\\\ZendAcl' => 'Acl\\\\LaminasAcl',
```

Comment:
Why do we need double namespace separator?

Reply:
Probably overkill, but was doing it to mimic the namespace entries. But you're right - for this, not needed.